File: linearsearch.py
def linearsearch(arr,key):
    if(len(arr)==0):
         return -1
    for i in range(len(arr)):
        if(arr[i]==key):
            return i
    return -1
x=int(input())
li=list(map(int,input().split()))
result=linearsearch(li,x)
if(result!=-1):
    print(f"The element found at the index {result}")
else:
    print("Element not found")
        

File: linearsearch.js
function linearsearch(arr,x){
if(arr.length==0){
    return -1
}
for(i=0;i<arr.length;i+=1){
    if(arr[i]==x){
        return i
    }
}
return -1
}
let x=5
let arr=[1,2,3,4,5]
console.log(linearsearch(arr,x))

